src/java/org/apache/cassandra/service/accord/AccordCallback.java

```diff
@@ -46,7 +46,7 @@ public AccordCallback(AgentExecutor executor, Callback<T> callback, AccordEndpoi
     @Override
     public void onResponse(Message<T> msg)
     {
-        logger.debug("Received response {} from {}", msg.payload, msg.from());
+//        logger.debug("Received response {} from {}", msg.payload, msg.from());
```

Comment:
will add back, way too much spam in the test logs making it hard to find *real* issues...